### PR TITLE
UI tweaks for Approaches

### PR DESCRIPTION
**Pull request by dem4ron**
### UI Changes:
Requested here: https://github.com/exercism/project-management/issues/52

#### New AvatarGroup component:

Screenshot 2022-10-10 at 12 36 51

Reworked Credits component, with AvatarGroup, easily configurable top label, bottom label of texts. 

#### Reworked fallback components:

Screenshot 2022-10-10 at 15 29 38

#### Video modal max width:

Screenshot 2022-10-10 at 16 02 22

### CODE REVIEW

File: app/javascript/components/track/approaches-elements/DiggingDeeper.tsx [hunk 1]
```diff
@@ -22,15 +22,21 @@ export function DiggingDeeper({
...
-      <section className="shadow-lgZ1 !py-[18px] mb-16 rounded-8 px-20 lg:px-32 py-20 lg:py-24">
-        <h2 className="mb-8 text-h2">Digging deeper</h2>
-        <div
-          className="c-textual-content --small"
-          dangerouslySetInnerHTML={{ __html: introduction.html }}
-        />
-      </section>
+      {introduction.html.length > 0 ? (
+        <>
+          <section className="shadow-lgZ1 !py-[18px] mb-16 rounded-8 px-20 lg:px-32 py-20 lg:py-24">
```
Comment (iHiD) [inline app/javascript/components/track/approaches-elements/DiggingDeeper.tsx]: @dem4ron:
1. This needs a `bg-white` to look right. 
2. I don't like the important flag here - why do we have that? 
3. We should stick to the base we're using, so make this `py-20` and shout at @taiyab about it :)

File: app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export function CommunityVideo({
             alt="thumbnail"
           />
           <div className="flex flex-col">
-            <h5 className="text-h5 mb-8">{video.title}</h5>
+            <h5 className="text-h5 mb-8 text-left">{video.title}</h5>
```
Comment (iHiD) [inline app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx]: @dem4ron Here's a loom on this bit of code in general: https://www.loom.com/share/0901d0e13a4242cca67ccd15a17ced93?from_recorder=1&focus_title=1

File: app/javascript/components/track/approaches-elements/no-content-yet/NoIntroductionYet.tsx [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
-      <a className="flex" href={introduction.links.new}>
-        <span className="underline">You can do it here.</span>&nbsp;
-        <Icon
-          className="filter-textColor6"
-          icon={'new-tab'}
-          alt={'open in a new tab'}
-        />
-      </a>
-    </NoContentYet>
+    <section className="shadow-lgZ1 !py-[18px] mb-16 rounded-8 px-20 lg:px-32 py-20 lg:py-24">
```
Comment (iHiD) [inline app/javascript/components/track/approaches-elements/no-content-yet/NoIntroductionYet.tsx]: Same comments here as above.

File: app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx (hunk 2 above)
Comment (dem4ron) [reply app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx]: Yeah, I agree, this was a mess.. I started using grid more to flatten things out, but that can become overcomplicated in Tailwind without template-areas support..

**Pull request merged**